qml/keys/NumKey.qml

```diff
@@ -100,12 +102,20 @@ Item {
 
     Text {
         id: keyLabel
-        text: (panel.activeKeypadState === "NORMAL") ? label : shifted;
-        anchors.centerIn: parent
+        //text: (panel.activeKeypadState === "NORMAL") ? label : shifted;
+		text: label
```

Comment:
Check the behavior ;) The text and annotation stay the same, they just change color + size so needed to deal with that ;)